### ISSUE TITLE
Build fails with sundials 6.3.0

### ISSUE THREAD

**Issue by onny** [bmcage/odes#138]
Hey,
unfortunately build fails with sundials 6.3.0 on NixOS. I get following build error:
```
scikits/odes/sundials/cvode.c: In function ‘__pyx_f_7scikits_4odes_8sundials_5cvode_5CVODE__reinit_IC’:
scikits/odes/sundials/cvode.c:31664:24: error: too few arguments to function ‘N_VMake_Serial’
31664 |     __pyx_v_self->y0 = N_VMake_Serial(__pyx_v_N, ((realtype *)__pyx_v_y0->data));
      |  
```

Regards
Jonas

**Comment by aragilar** [comment]
As first glance, it looks like sundials has changed the function signature in a backwards incompatible way. If you want to make a PR to fix it, that would be helpful.

**Comment by aragilar** [comment]
Should be fixed by #144.

**Comment by onny** [comment]
Thank you, works now for us on NixOS :)
